services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/appconfig/YamlLoaderTest.java

```diff
@@ -40,7 +40,8 @@ void okFile() throws UnableToLoadFileException {
   @ValueSource(strings = {
       "configtests/app-config_empty.yaml",
       "configtests/wrong_file.yaml",
-      "configtests/broken_syntax.yaml",
+      "configtests/app-config_broken_syntax.yaml",
+      "parameters/naming_mismatch.yaml",
```

Comment:
Would be nice to have the test files organized a bit more "cleanly". Looks like files belonging to similar test cases are scattered across multiple different directories.